dc/core/Markers.h

```diff
@@ -46,27 +46,37 @@
 
 #include <QObject>
 #include <QPointF>
+#include <QtCore/QAbstractListModel>
 #include <map>
 
 #include <boost/serialization/access.hpp>
 #include <boost/enable_shared_from_this.hpp>
 #include <boost/serialization/map.hpp>
 
-typedef std::map<int, QPointF> MarkersMap;
+typedef std::pair<int, QPointF> Marker;
+typedef std::vector<Marker> MarkersVector;
```

Comment:
typedefs can be private in class, not used outside the class from what I can see